cohorts/functions.py

```diff
@@ -84,7 +84,8 @@ def count_filter_fn(filterable_variant, **kwargs):
             return ((filterable_variant_function(filterable_variant) if filterable_variant_function is not None else True) and
                     filter_fn(filterable_variant, **kwargs))
         patient_id = row["patient_id"]
-        return cohort.load_variants(
+        return cohort.load_effects(
```

Comment:
I see that you wrote:

> Finally, to make this work relatively seamlessly, I had to modify count_variants_function_builder to instead load effects -- this way any future filtering function wouldn't have to know if it's working on a variant or an effect. Not sure what downstream implications this might have for validity/performance.

But I'm not clear what `count_effects_function_builder` doesn't provide, and still am confused about why we're removing the `variant` counter?

You probably already know this, but the same `filter_fn` can be passed into `load_effects` and `load_variants`, since a `FilterableEffect` is a subclass of a `FilterableVariant` (and similar stuff for `FilterableNeoantigen` etc.)

All that to say: I'm just a bit confused :)